repondeur/zam_repondeur/fetch/senat/amendements.py

```diff
@@ -103,15 +103,25 @@ def parse_from_csv(row: dict, lecture: Lecture) -> Tuple[Amendement, bool]:
         amendement.article = article
     amendement.rectif = rectif
     amendement.alinea = row["Alinéa"].strip()
-    amendement.auteur = row["Auteur "]
+    amendement.auteur = extract_auteur(row)
+
     amendement.matricule = extract_matricule(row["Fiche Sénateur"])
+
     amendement.date_depot = parse_date(row["Date de dépôt "])
     amendement.sort = row["Sort "]
     amendement.dispositif = clean_html(row["Dispositif "])
     amendement.objet = clean_html(row["Objet "])
     return amendement, created
 
 
+def extract_auteur(row: dict) -> str:
+    auteur: str = row["Auteur "]
+    au_nom_de = row.get("Au nom de ", "")
+    if au_nom_de.startswith("commission"):
+        auteur += f", au nom de la {au_nom_de}"
```

Comment:
Given the size, it will probably break something on the design/front side.

Reply:
Yes, let’s talk about it with the team